src/client/components/Analytics/index.js

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect, useState } from 'react';
+import { Table } from '@instructure/ui-table';
+import axios from 'axios';
+import { ltikPromise } from '../../services/ltik';
+
+const Analytics = () => {
+  const [analytics, setAnalytics] = useState([]);
+
+  const getAnalytics = () => {
+    ltikPromise.then(ltik => {
+      axios.get(`/api/getanalytics?ltik=${ltik}`).then(res => {
+        setAnalytics(res.data);
+      });
+    });
+  };
+
+  useEffect(getAnalytics, []);
+  console.log(analytics);
+  const analyticsBody = analytics
+    ? analytics.map(stat => (
+        <Table.Row>
+          <Table.RowHeader>{stat.contextId}</Table.RowHeader>
+          <Table.Cell>{stat.shortname}</Table.Cell>
+          <Table.Cell>{stat.numMembers}</Table.Cell>
+          <Table.Cell>{stat.total_reserve_clicks}</Table.Cell>
+          <Table.Cell>
+            {((stat.reserve_clicks * 100) / stat.numMembers).toPrecision(3)}
```

Comment:
@finnzink You are doing the same operation here multiple times.. Why not just create a new component or function that will format the number for you? Keep in mind: https://en.wikipedia.org/wiki/Don%27t_repeat_yourself